app/assets/stylesheets/frontend.less

```diff
@@ -16,34 +16,43 @@ body {
 	font-family: sans-serif;
 }
 
-.section_headings {
-    position: relative;
-    z-index: 1;
-    overflow: hidden;
-    text-align: center;
+h1 {
+	/* Page itself has top padding */
+	margin-top: 0
 }
 
-.section_headings:before, .section_headings:after {
-    position: absolute;
-    top: 51%;
-    overflow: hidden;
-    width: 50%;
-    height: 1px;
-    content: '\a0';
-    background-color: black;
+h1, h2 {
+	margin-bottom: 20px;
 }
 
-.section_headings:before {
-    margin-left: -51%;
-    text-align: right;
+h2 {
+	border-top: 1px solid #dddddd;
+	padding-top: 8px;
+	page-break-before: always
 }
 
-.section_headings:after {
-    margin-left: 1%;
+h1 + h2 {
+	page-break-before: avoid
 }
 
-h1, h2 {
-    margin-bottom: 20px;
+h1, h2, h3, h4, h5, h6 {
+	page-break-after: avoid
+}
+
+h3 {
+	border-top: 1px solid #eeeeee;
```

Comment:
Mixing centred headings with left-aligned text is [fairly widely](http://uxmovement.com/content/why-you-should-never-center-align-paragraph-text/) [agreed to](http://www.printwand.com/blog/basic-alignment-principles-in-graphic-design-with-examples) [be a bad plan](http://www.visualmess.com/#sec4.1).

I think it’s typically fine to have the `h1` be centred, but it’s best to avoid switching between centred (headings) and left-aligned (paragraph) text throughout a document.

Overall, I think the most important part of this commit was the whitespace changes. The page breaks are orthogonal, and I will split the commit and resubmit it separately.